docs/content/doc/advanced/customizing-gitea.en-us.md

```diff
@@ -88,6 +88,8 @@ just place it under your "custom/public/" directory (for instance `custom/public
 To match the current style, the link should have the class name "item", and you can use `{{AppSubUrl}}` to get the base URL:
 `<a class="item" href="{{AppSubUrl}}/impressum.html">Impressum</a>`
 
+For more information, see [Adding Legal Pages](/en-us/adding-legal-pages).
```

Comment:
links in others parts of docs seems to not be relative.
```suggestion
For more information, see [Adding Legal Pages](https://docs.gitea.io/en-us/adding-legal-pages).
```

Reply:
@sapk Hmm... I think you misunderstood relative and absolute link. Your suggestion actually use absolute link (there is `docs.gitea.io` domain in the URL), while I use relative link (from docs root).

Reply:
BTW, I will apply it.